### DISCUSSION TITLE
Disable containerized analytics collection?

### DISCUSSION THREAD

**Discussion by emperorkebab** [zitadel/zitadel#7994] [Q&A] [answered]
Hi i have a question about usage data (analytics) collection.
The readme part talking about this was removed in this PR https://github.com/zitadel/zitadel/pull/3978 and merged in this commit https://github.com/zitadel/zitadel/commit/aed70105083c888202cfe796edd7d5a8e8cd0861

Is Zitadel collecting usage data in the current versions?, and if so how to disable it in a containerized environment (k8s, docker, how to use the `--disable-analytics` flag?)

**Comment by hifabienne** [comment, marked as answer]
We do not collect any usage data at the moment.
